tests/view/items/test_items.py

```diff
@@ -0,0 +1,71 @@
+""" Tests for the items.
+"""
+from http import HTTPStatus
+from datetime import datetime
+
+from tests.view.items.mock_items import ItemMock, ItemsMock
+from timeless.items.views import ItemListView
+
+
+def test_list(client):
+    """ Test list is okay """
+    items = [
+        {
+            "id": 1,
```

Comment:
@paulodamaso Look at this example
https://github.com/timelesslounge/timelessis/pull/395#discussion_r259639114

We have a factory lib which provide a mechanism to create fake data and helps us to avoid a lot of copy/pate code like this.

If you have questions, pls aks me.

Reply:
@vladarefiev Using that factory caused me some problems in other tickets, so I've decided to make this one manually. On top of this I think that we have more control of our tests if we use "real" data to them instead just some text being repeated.

Reply:
@paulodamaso add here @emilianodellacasa let's decide

Reply:
@paulodamaso Inside `ListView` we fetch data from DB. It's how it works. So, it's intergation test by default.

Reply:
@vladarefiev @paulodamaso Let's use factories